comp-sci/data-structures-and-algorithms/intro-data-structures/the-queue-data-structure.md

```diff
@@ -88,4 +88,4 @@ The working principle behind a queue is
 ## Footnotes
 
 [1:ADT]
-This is just an abreviation for the Abstract Data Structure, hence ADT.
\ No newline at end of file
+Abstract Data Type, or ADT, means the data type cannot be implemented (i.e. represented) by itself and it needs some kind of 'support' in order to be used.
```

Comment:
Not sure if this is clear enough 🤔 

"abstract" in CS usually means that you explain that something will have certain capabilities but not the underlying implementation details.

How about we say this?

```suggestion
Abstract Data Type, or ADT, is a definition of a data type (or a data structure) that only mentions what operations are to be performed but not how these operations will be implemented.
```

Reply:
yeah, it sounds much better.

we also have a footnote on ADT in `The Linked List Data Structure` insight. should I update that one as well?

Reply:
Yeah, let's do it

Reply:
okay, I'll put both updates in this pr